styles.css

```diff
@@ -60,44 +86,44 @@ Useful tools:
 */
 
 .btn {
-  background:#fff;
-  border:.05rem solid #218a39;                  /* primary */
-  color:#1a6d2d;                                /* primary */
-}
-
-.btn:focus {
-  box-shadow:0 0 0 .1rem rgba(33, 138, 57, .2)  /* primary in RGB */
+  background: #f2f2f2;
+  border:.05rem solid #999999;                  /* primary */
+  border-radius: 0.2rem;
+  color: #666666;                                /* primary */
 }
 
 .btn:focus,
 .btn:hover {
-  background:#d0f4d8;                           /* primary lighten 37 */
-  border-color:#1e7e34;                         /* primary darken 3 */
+  background: #666666;                           /* primary lighten 37 */
```

Comment:
Comments do not match the code any longer.

Reply:
Removed the comments from button stylings.